app/src/Reservations/Reservations.js

```diff
@@ -1,9 +1,66 @@
 import React from "react";
 
+import * as apiClient from "../apiClient";
+import Border from "../images/Border.png";
+
+import CurrentWeather from "./CurrentWeather";
+import ForecastWeather from "./ForecastWeather";
+import styles from "./styles.module.scss";
+
 function Reservations() {
+  const [currentWeather, setCurrentWeather] = React.useState({});
+  const [forecastWeather, setForecastWeather] = React.useState({});
+  const [units, setUnits] = React.useState("F");
+  const [fiveDayForecast, setFiveDayForecast] = React.useState(false);
+
+  const handleForecastButton = (e) => {
+    return fiveDayForecast
+      ? setFiveDayForecast(true)
+      : setFiveDayForecast(false);
+  };
+
+  const loadCurrentWeather = async () =>
+    setCurrentWeather(await apiClient.getCurrentWeather());
```

Comment:
I got a recommendation from my mentor, to create different API clients for each route. You can do that in the future. For example, one for weather, another one for your menu data from the back-end and so on